spec/priorityQueue.js

```diff
@@ -0,0 +1,26 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import pQueue from '../src/PriorityQueue'
+
+chai.use(chaiChange)
+
+describe('pQueue', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(pQueue).to.be.a('function')
+  })
+ context('enqueue', () => {
+   it(' adds an element with priority (number) to the back of the queue.',() => {
+      const myPqueue = new  Pqueue
+
+    expect(() => myPqueue.push('foo')
+    .to.alter(() => myPqueue.length(), { from:0 , to:1})
+  })
+ })
```

Comment:
Make sure to indent to format. That will make it way easier to see when you may have missing or extra parentheses and braces.